src/goto-programs/remove_function_pointers.cpp

```diff
@@ -451,6 +517,7 @@ void remove_function_pointerst::remove_function_pointer(
     });
 }
 
+
```

Comment:
No need for all these added blanks, convention in the codebase seems to be one blank line between functions